[2024-07-14T04:20:59.097+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-14T04:20:59.222+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ELT-PROJECT.transform.dim_employees.test scheduled__2024-07-09T00:00:00+00:00 [queued]>
[2024-07-14T04:20:59.344+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ELT-PROJECT.transform.dim_employees.test scheduled__2024-07-09T00:00:00+00:00 [queued]>
[2024-07-14T04:20:59.353+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-07-14T04:20:59.564+0000] {taskinstance.py:2330} INFO - Executing <Task(DbtTestLocalOperator): transform.dim_employees.test> on 2024-07-09 00:00:00+00:00
[2024-07-14T04:20:59.700+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=16203) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-07-14T04:20:59.725+0000] {standard_task_runner.py:63} INFO - Started process 16230 to run task
[2024-07-14T04:20:59.743+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'ELT-PROJECT', 'transform.dim_employees.test', 'scheduled__2024-07-09T00:00:00+00:00', '--job-id', '1277', '--raw', '--subdir', 'DAGS_FOLDER/elt.py', '--cfg-path', '/tmp/tmp7desjii1']
[2024-07-14T04:20:59.777+0000] {standard_task_runner.py:91} INFO - Job 1277: Subtask transform.dim_employees.test
[2024-07-14T04:20:59.874+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-07-14T04:21:00.183+0000] {task_command.py:426} INFO - Running <TaskInstance: ELT-PROJECT.transform.dim_employees.test scheduled__2024-07-09T00:00:00+00:00 [running]> on host airflow-scheduler
[2024-07-14T04:21:00.983+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Data-Ninja' AIRFLOW_CTX_DAG_ID='ELT-PROJECT' AIRFLOW_CTX_TASK_ID='transform.dim_employees.test' AIRFLOW_CTX_EXECUTION_DATE='2024-07-09T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-09T00:00:00+00:00'
[2024-07-14T04:21:00.987+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-14T04:21:00.990+0000] {local.py:174} INFO - Could not import dbtRunner. Falling back to subprocess for invoking dbt.
[2024-07-14T04:21:01.006+0000] {local.py:292} INFO - Cloning project to writable temp directory /tmp/tmpd614_7hn from /opt/airflow/dags/dbt/dbt_project
[2024-07-14T04:21:01.018+0000] {local.py:303} INFO - Partial parse is enabled and the latest partial parse file is /tmp/cosmos/ELT-PROJECT__transform/target/partial_parse.msgpack
[2024-07-14T04:21:01.204+0000] {base.py:84} INFO - Using connection ID 'snow_conn' for task execution.
[2024-07-14T04:21:01.209+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2024-07-14T04:21:01.228+0000] {config.py:298} INFO - Profile caching is enable.
[2024-07-14T04:21:01.243+0000] {config.py:272} INFO - Profile found in cache using profile: /tmp/cosmos/profile/a0017dab0633177292578162811b91dcb9c56d6e54017e9660056cc364a762ad/profiles.yml.
[2024-07-14T04:21:01.253+0000] {local.py:247} INFO - Trying to run the command:
 ['/opt/airflow/dbt_venv/bin/dbt', 'deps', '--project-dir', '/tmp/tmpd614_7hn', '--profiles-dir', '/tmp/cosmos/profile/a0017dab0633177292578162811b91dcb9c56d6e54017e9660056cc364a762ad', '--profile', 'dbt_project', '--target', 'dev']
From /tmp/tmpd614_7hn
[2024-07-14T04:21:01.257+0000] {subprocess.py:60} INFO - Tmp dir root location: 
 /tmp
[2024-07-14T04:21:01.269+0000] {subprocess.py:73} INFO - Running command: ['/opt/airflow/dbt_venv/bin/dbt', 'deps', '--project-dir', '/tmp/tmpd614_7hn', '--profiles-dir', '/tmp/cosmos/profile/a0017dab0633177292578162811b91dcb9c56d6e54017e9660056cc364a762ad', '--profile', 'dbt_project', '--target', 'dev']
[2024-07-14T04:21:01.377+0000] {subprocess.py:84} INFO - Command output:
[2024-07-14T04:21:21.375+0000] {subprocess.py:94} INFO - [0m04:21:21  Running with dbt=1.8.3
[2024-07-14T04:21:22.652+0000] {subprocess.py:94} INFO - [0m04:21:22  Updating lock file in file path: /tmp/tmpd614_7hn/package-lock.yml
[2024-07-14T04:21:22.679+0000] {subprocess.py:94} INFO - [0m04:21:22  Installing dbt-labs/dbt_utils
[2024-07-14T04:21:25.202+0000] {subprocess.py:94} INFO - [0m04:21:25  Installed from version 0.8.6
[2024-07-14T04:21:25.222+0000] {subprocess.py:94} INFO - [0m04:21:25  Updated version available: 1.2.0
[2024-07-14T04:21:25.252+0000] {subprocess.py:94} INFO - [0m04:21:25
[2024-07-14T04:21:25.273+0000] {subprocess.py:94} INFO - [0m04:21:25  Updates available for packages: ['dbt-labs/dbt_utils']
[2024-07-14T04:21:25.284+0000] {subprocess.py:94} INFO - Update your versions in packages.yml, then run dbt deps
[2024-07-14T04:21:28.201+0000] {subprocess.py:98} INFO - Command exited with return code 0
[2024-07-14T04:21:28.203+0000] {local.py:254} INFO - Update your versions in packages.yml, then run dbt deps
[2024-07-14T04:21:28.206+0000] {local.py:247} INFO - Trying to run the command:
 ['/opt/airflow/dbt_venv/bin/dbt', 'test', '--models', 'dim_employees', '--project-dir', '/tmp/tmpd614_7hn', '--profiles-dir', '/tmp/cosmos/profile/a0017dab0633177292578162811b91dcb9c56d6e54017e9660056cc364a762ad', '--profile', 'dbt_project', '--target', 'dev']
From /tmp/tmpd614_7hn
[2024-07-14T04:21:28.209+0000] {subprocess.py:60} INFO - Tmp dir root location: 
 /tmp
[2024-07-14T04:21:28.211+0000] {subprocess.py:73} INFO - Running command: ['/opt/airflow/dbt_venv/bin/dbt', 'test', '--models', 'dim_employees', '--project-dir', '/tmp/tmpd614_7hn', '--profiles-dir', '/tmp/cosmos/profile/a0017dab0633177292578162811b91dcb9c56d6e54017e9660056cc364a762ad', '--profile', 'dbt_project', '--target', 'dev']
[2024-07-14T04:21:28.278+0000] {subprocess.py:84} INFO - Command output:
[2024-07-14T04:21:44.639+0000] {subprocess.py:94} INFO - [0m04:21:44  Running with dbt=1.8.3
[2024-07-14T04:21:49.285+0000] {subprocess.py:94} INFO - [0m04:21:49  Registered adapter: snowflake=1.8.3
[2024-07-14T04:21:54.547+0000] {subprocess.py:94} INFO - [0m04:21:54  Found 20 models, 27 data tests, 11 sources, 684 macros
[2024-07-14T04:21:54.612+0000] {subprocess.py:94} INFO - [0m04:21:54
[2024-07-14T04:22:01.336+0000] {subprocess.py:94} INFO - [0m04:22:01  Concurrency: 1 threads (target='dev')
[2024-07-14T04:22:01.342+0000] {subprocess.py:94} INFO - [0m04:22:01
[2024-07-14T04:22:01.436+0000] {subprocess.py:94} INFO - [0m04:22:01  1 of 2 START test not_null_dim_employees_EMPLOYEEID ............................ [RUN]
[2024-07-14T04:22:03.648+0000] {subprocess.py:94} INFO - [0m04:22:03  1 of 2 PASS not_null_dim_employees_EMPLOYEEID .................................. [[32mPASS[0m in 2.21s]
[2024-07-14T04:22:03.664+0000] {subprocess.py:94} INFO - [0m04:22:03  2 of 2 START test unique_dim_employees_EMPLOYEEID .............................. [RUN]
[2024-07-14T04:22:05.885+0000] {subprocess.py:94} INFO - [0m04:22:05  2 of 2 PASS unique_dim_employees_EMPLOYEEID .................................... [[32mPASS[0m in 2.21s]
[2024-07-14T04:22:05.916+0000] {subprocess.py:94} INFO - [0m04:22:05
[2024-07-14T04:22:05.920+0000] {subprocess.py:94} INFO - [0m04:22:05  Finished running 2 data tests in 0 hours 0 minutes and 11.30 seconds (11.30s).
[2024-07-14T04:22:06.442+0000] {subprocess.py:94} INFO - [0m04:22:06
[2024-07-14T04:22:06.450+0000] {subprocess.py:94} INFO - [0m04:22:06  [32mCompleted successfully[0m
[2024-07-14T04:22:06.454+0000] {subprocess.py:94} INFO - [0m04:22:06
[2024-07-14T04:22:06.458+0000] {subprocess.py:94} INFO - [0m04:22:06  Done. PASS=2 WARN=0 ERROR=0 SKIP=0 TOTAL=2
[2024-07-14T04:22:09.955+0000] {subprocess.py:98} INFO - Command exited with return code 0
[2024-07-14T04:22:09.957+0000] {local.py:254} INFO - [0m04:22:06  Done. PASS=2 WARN=0 ERROR=0 SKIP=0 TOTAL=2
[2024-07-14T04:22:10.784+0000] {local.py:196} WARNING - Artifact schema version: https://schemas.getdbt.com/dbt/manifest/v12.json is above dbt-ol supported version 7. This might cause errors.
[2024-07-14T04:22:10.786+0000] {local.py:196} WARNING - Artifact schema version: https://schemas.getdbt.com/dbt/run-results/v6.json is above dbt-ol supported version 5. This might cause errors.
[2024-07-14T04:22:10.863+0000] {local.py:349} INFO - Inlets: [Dataset(uri='snowflake://OU00370.europe-west2.gcp/ELT_BATCH.dwh.dim_employees', extra=None)]
[2024-07-14T04:22:10.903+0000] {local.py:350} INFO - Outlets: []
[2024-07-14T04:22:10.909+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-07-14T04:22:11.300+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-14T04:22:11.450+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=ELT-PROJECT, task_id=transform.dim_employees.test, run_id=scheduled__2024-07-09T00:00:00+00:00, execution_date=20240709T000000, start_date=20240714T042059, end_date=20240714T042211
[2024-07-14T04:22:12.538+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-14T04:22:12.630+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
